lib/graphql/schema/default_renamer.rb

```diff
@@ -0,0 +1,13 @@
+module GraphQL
+  class Schema
+    module DefaultRenamer
```

Comment:
What are the performance costs of dispatching a method for renaming? I would imagine not much; but it would be good to know.